kolibri/core/templatetags/kolibri_tags.py

```diff
@@ -36,15 +39,51 @@ def kolibri_main_navigation():
 
 def render_as_tags(bundle):
     tags = []
-    static = getattr(settings, 'STATIC_URL')
     for chunk in bundle:
-        url = chunk.get('publicPath') or chunk['url']
         if chunk['name'].endswith('.js'):
-            tags.append('<script type="text/javascript" src="{static}/{url}"></script>'.format(static=static, url=url))
+            tags.append('<script type="text/javascript" src="{url}"></script>'.format(url=render_as_url(chunk)))
```

Comment:
@rtibbles clarified: the loaded JS is wrapped by `webpack` and not in the global namespace. However it is true that e.g. `module.exports = exampleModule;` has no effect, and all communication must happen through events with typical plugins rather than exported modules like in CommonJS and AMD.